03_Uitvoering.Rmd

```diff
@@ -1971,16 +1983,16 @@ if (nrow(tubes_cat3_bis %>% filter(selectie == 0)) > 0 | nrow(tubes_cat2_bis %>%
 if (nognietklaar == TRUE) {
   export <- bind_rows(
     tubes_cat3_bis %>% mutate (cat = 3) %>% 
-      select(loc_code, nryears, firstyear, lastyear, selectie, cat) , 
+      dplyr::select(loc_code, nryears, firstyear, lastyear, selectie, cat) , 
     tubes_cat2_bis %>% mutate (cat = 2) %>%  
-      select(loc_code, nryears, firstyear, lastyear, selectie, cat) , 
+      dplyr::select(loc_code, nryears, firstyear, lastyear, selectie, cat) , 
   ) %>% 
     filter(selectie == 0)  
   kobj <- kable( bind_rows(
         tubes_cat3_bis %>% 
-          select(loc_code, nryears, firstyear, lastyear, selectie) , 
+          dplyr::select(loc_code, nryears, firstyear, lastyear, selectie) , 
         tubes_cat2_bis %>% 
-          select(loc_code, nryears, firstyear, lastyear, selectie) , 
+          dplyr::select(loc_code, nryears, firstyear, lastyear, selectie) , 
```

Comment:
I wonder why you added `dplyr::` everywhere. Was `select()` masked by another package? In such case, the order of loading packages in setup.Rmd is the key.

But of course it's safe to keep it this way.

Reply:
I know but at a given moment the R-code was interrupted because of the wrong select-function (also pull is a delicate one)  and I didn't want to start all over again. 
It's safer, but probably redundant.

Reply:
I think the conflicts are due to `raster::select()` and `git2r::pull()`. If you load dplyr or tidyverse at the end of chunk library(...) statements in index.Rmd, you shouldn't have problems (then dplyr::xxx will mask the other ones).